package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "arc_hive",
+  "version": "1.0.0",
+  "description": "Slack app used for cataloging lecture videos",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cs1Capstone/arc_hive.git"
+  },
+  "author": "Evan Allen, Tyge Johnson, Patrick Kennedy, Ryan Sherrill, Lois Truby",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/cs1Capstone/arc_hive/issues"
+  },
+  "homepage": "https://github.com/cs1Capstone/arc_hive#readme",
+  "dependencies": {
+    "body-parser": "^1.18.2",
+    "express": "^4.16.2",
+    "mongoose": "^4.13.5",
```

Comment:
Mongo may still make sense for some of your data persisting, but I also encourage checking out Airtable - https://www.npmjs.com/package/airtable

Sorry I've not given specifics yet, I'll try to get a copy of the Airtable base we're actually using and share it with you soon.